src/sb-build-test-bundlers

```diff
@@ -18,7 +18,7 @@ var es6 = function(bundler) {
     '/**',
     ' * ' + bundler + ' test ',
     ' */',
-    'import pkg from "' + path.join('..', '..', path.relative(config.path, config.jsNextMain)) + '";',
+    'import pkg from "' + path.join('..', '..', path.relative(config.path, config.jsNextMain)).replace(/\\/g,'/') + '";',
```

Comment:
Another option, since we want to always have forward slashes here, we can do `path.posix.join` and `path.posix.relative` and it'll make sure it uses posix separators.